src/app/legacy/containers/SocialEmbed/sourceHelpers.js

```diff
@@ -36,7 +36,7 @@ export const getIdFromSource = source => {
   const sourceIds = {
     twitter: /\/status\/([0-9]+)/,
     youtube: /\/watch\?v=([0-9 A-Z a-z _-]+)/,
-    instagram: /\/p\/([0-9 A-Z a-z]+)/,
+    instagram: /\/p\/([0-9 A-Z a-z _-]+)/,
```

Comment:
this pattern also allows spaces (as before and as do the others). It that deliberate style choice?

Reply:
No particular reason for this other than following what was in the others. I'll remove the spacing though as there shouldn't be any spaces allowed in these URLs.